.azure-pipelines/ci-build.yml

```diff
@@ -81,11 +81,18 @@ extends:
               projects: '$(Build.SourcesDirectory)\Microsoft.OpenApi.OData.sln'
               arguments: '--configuration $(BuildConfiguration) --no-build'
 
-          - task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@3
+          - task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@5
             displayName: 'ESRP CodeSigning'
             inputs:
-              ConnectedServiceName: 'microsoftgraph ESRP CodeSign DLL and NuGet (AKV)'
-              FolderPath: src
+              ConnectedServiceName: 'Federated DevX ESRP Managed Identity Connection'
+              FolderPath: '$(Build.SourcesDirectory)\src'
+              AppRegistrationClientId: '65035b7f-7357-4f29-bf25-c5ee5c3949f8'
+              AppRegistrationTenantId: 'cdc5aeea-15c5-4db6-b079-fcadd2505dc2'
+              AuthAKVName: 'akv-prod-eastus'
+              AuthCertName: 'ReferenceLibraryPrivateCert'
+              AuthSignCertName: 'ReferencePackagePublisherCertificate'              
+              Pattern: '*.dll'
```

Comment:
Double check this. Pipeline reports this and dlls are note signed.
`##[warning] No files found to sign. You must double check your configuration for input file names or minimatch pattern *.dll to ensure they are correct. -- Warning`

Reply:
Fixed this ✅